project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject amperity/vault-clj "0.3.5-SNAPSHOT"
+(defproject amperity/vault-clj "0.5.0-SNAPSHOT"
```

Comment:
Hmm, I would still go with either `0.4.0` or `1.0.0`. I think jumping a minor version would just be confusing, as it would look like there were past releases that were lost or deliberately erased.